### ISSUE TITLE
pnpm resolves msw optional dependency as v2 instead of v1

### ISSUE THREAD

**Issue by LavaToaster** [mswjs/data#289]
Given the following package.json file:

```
{
    "dependencies": {
        "msw": "^1.2.3"
    },
    "devDependencies": {
        "@mswjs/data": "^0.13.0",
        "@mswjs/http-middleware": "^0.8.0",
        "@mswjs/interceptors": "^0.25.1"
    }
}
```

NPM, Yarn and PNPM are resolving the `msw` for `@mswjs/data` as either v1 or v2. In our case with PNPM it's resolving v2. Which then results in @mswjs/data being incompatible.

Should this be a `peerDependency`?

**Comment by VororRich** [comment]
Should package.json have the devDependencies and optionalDependencies for MSW set to "^1.0.0" (instead of "latest") until a v2 fix/implementation is in place?

**Comment by pim** [comment]
In the meantime adding this to package.json fixed it for me.

```
{
  "pnpm": {
    "overrides": {
      "@mswjs/data": "^1.0.0"
    }
}
```

**Comment by kettanaito** [comment]
Using `latest` was completely incorrect. Will publish a fix.

**Comment by kettanaito** [comment]
## Released: v0.15.0 🎉

This has been released in v0.15.0!

- 📄 [**Release notes**](https://github.com/mswjs/data/releases/tag/v0.15.0)
- 📦 [npm package](https://www.npmjs.com/package/@mswjs/data/v/0.15.0)

Make sure to always update to the latest version (`npm i @mswjs/data@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
